apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -109,7 +111,7 @@ public static ComponentsDefine getInstance() {
     }
 
     public ComponentsDefine() {
-        components = new String[50];
+        components = new String[100];
```

Comment:
Why 100？

Reply:
I refer to ArrayList's scaling, double the size each time.
It's ok if change it to 51 or another smaller one.

Reply:
I think it isn't big problem. I just ask.